### PR TITLE
Add internal hm9000 stop controller

### PR DESCRIPTION
**Pull request by ScarletTanager**
We are converting hm9000 to use http rather than NATS for hm9000.stop messages.  This adds the required controller into the CC.

* [X] I have viewed signed and have submitted the Contributor License Agreement

* [X] I have made this pull request to the `master` branch

* [X] I have run all the unit tests using `bundle exec rake`

* [X] I have run CF Acceptance Tests on bosh lite

[#112003161]

Signed-off-by: Jonathan Berkhahn <jaberkha@us.ibm.com>

### CODE REVIEW

File: spec/unit/lib/cloud_controller/dea/hm9000/respondent_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+module VCAP::CloudController
+  module Dea
+    module HM9000
+      describe Respondent do
+        let(:message_bus) { CfMessageBus::MockMessageBus.new }
+        let(:dea_client) { double(VCAP::CloudController::Dea::Client) }
+
+        subject(:respondent) { Dea::HM9000::Respondent.new(dea_client, message_bus) }
+
+        # Fill this in
```
Comment (adowns01) [inline spec/unit/lib/cloud_controller/dea/hm9000/respondent_spec.rb]: Did you intend to add tests here?

**Comment by adowns01** [comment]
Hello @ScarletTanager, 

It looks good except an abandoned comment about a possible additional test? Should the comment be deleted or is there a test you want to add? 

Everything looks good on our end. Will merge when you reply and fix.

Thanks, 
@adowns01 && @rizwanreza, CAPI Team Members

**Comment by ScarletTanager** [comment]
Hi @adowns01 , I'll delete the comment ASAP.

**Comment by ScarletTanager** [comment]
Comment deleted.

**Comment by adowns01** [comment]
👍

**Pull request merged**